filebeat/module/zookeeper/audit/test/audit.log

```diff
@@ -0,0 +1,17 @@
+user=zookeeper/192.168.1.3 operation=serverStart   result=success
```

Comment:
@matschaffer it'd be nice to have your input also here, are you using zookeeper audit logging? do you configure it with a pattern that includes timestamps?

Reply:
ya i also thought the lack of timestamps was weird.  THe logs i used were straight from the examples in the docs https://zookeeper.apache.org/doc/r3.7.0/zookeeperAuditLogs.html even though now that i'm looking further down, the default audit logging config they show is `log4j.appender.RFAAUDIT.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n` so IDK.  I can go off of this pattern and update the sample audit logs.  Based on https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html, the logs shown could just by the `%m` field and they didn't show the rest

Reply:
I updated the audit log pipeline to add GROK potentially the "default" pattern above.  Now it will process that pattern or just the key value pairs directly.

Reply:
looking at https://github.com/apache/zookeeper/blob/master/conf/log4j.properties, i think we should be good now.